plutus-conformance/uplc/evaluation/exception/ifThenElse-no-force/ifThenElse-no-force.uplc.expected

```diff
@@ -0,0 +1,2 @@
+An error has occurred:  A builtin received a term argument when something else was expected
```

Comment:
I don't think our error outputs can be printing the actual error. Two reasons:
- Other tools may not even have the same evaluation model: there's no reason why they have to e.g. implement an evaluator as a CEK machine.
- Other tools may (reasonably) format their errors differently.

So I think for failures we may just have to record that it's a failure and nothing else, although that feels like it's throwing away information.

Reply:
Well, then what does the `.exception` file help with?

We could potentially include some non-normative information about the test case, where this info could potentially live, e.g.

```
out-of-scope-variable/README.md

This test checks that the evaluator fails when it encounters a variable that is not in scope. For example, the Haskell CEK machine will fail with the following error in this case:
<error here>
```
?

I'm just not sure that implementation-specific information like this belongs in a test suite where the whole point is to be implementation-independent!

Reply:
I was thinking the `.exception` files are for us to see what exception is thrown for the corresponding input files. It's not going to be tested against. Basically the same as your README idea above, but I'll make the executable write the files. I can name them README, that probably better.